Q.py

```diff
@@ -75,18 +75,41 @@ def get_next_action(boxes, epsilon, q_table):
             q_move[box] = max_move
 
         #get the box with the highest q value
-        print(q_max)
         box = max(q_max, key=q_max.get)
 
         #if the max q value is not 0, return the box and its move
         #else return the random box and move
         if q_max[box] != 0:
-            box_move = {box: q_move[box]}
+            box_move = [box, q_move[box]]
             return box_move
         else:
             return box_move
     else:
         return box_move
 
-def update_q_table(q_table, rewards):
+def update_q_table(q_table, rewards, new_box_position, action_taken, discount_factor, learning_rate):
```

Comment:
we only have one Q table per trial-- why is this class comprised of static functions?  Why not a q table object?